nextflow_schema.json

```diff
@@ -164,6 +164,11 @@
                     "description": "Run SPAdes/metaSPAdes without the error correction step.",
                     "default": true
                 },
+                "diamond_db":{
+                    "type": "string",
+                    "format": "file-path",
+                    "description":"Path to the Swiss-Prot or TREMBL db for diamond"
```

Comment:
This one is to check the quality of the assembly right? I would add some of that to the description; or would even create a new section in the config for post_assembly_qc

Reply:
You need to add `default = ""` to keep it consitent with the nextflow.config file ( the linter should have complained about that )

Reply:
I forgot to lint!!! :fear

Reply:
Will give a try to your new masterpiece